### PR TITLE
Feature/fma 47/vp support

### PR DESCRIPTION
**Pull request by zoemaas**

### CODE REVIEW

File: k8s/gx-portal-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
```
Comment (sksadjad) [inline k8s/gx-portal-deployment.yaml]: can you move this to gx-deployment project?

File: src/components/molecules/AssetTeaser.tsx [hunk 2]
```diff
@@ -25,8 +29,19 @@ const AssetTeaser: React.FC<AssetTeaserProps> = ({
...
+  const { attributes } = ddo.findServiceByType(
+    'metadata'
+  ) as ServiceMetadataMarket
+  const isCompliant = !!attributes.additionalInformation.compliance?.gx
   const { name, type } = attributes.main
+  // It shouldn't be undefined anymore, but maybe the best place to handle it would be in the input validation of the publish page
+  const sd = attributes.additionalInformation?.serviceSelfDescription
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  const legalName = sd?.raw
```
Comment (sksadjad) [inline src/components/molecules/AssetTeaser.tsx]: extract this in a util method

File: src/components/molecules/FormFields/ServiceSelfDescription.tsx [hunk 3]
```diff
@@ -40,13 +40,9 @@ export default function ServiceSelfDescription(
       props?.setStatus('loading')
 
       const parsedServiceSelfDescription = JSON.parse(rawServiceSelfDescription)
-      const signedServiceSelfDescription =
```
Comment (sksadjad) [inline src/components/molecules/FormFields/ServiceSelfDescription.tsx:43]: @nklomp do we want to delete the support for old structure here? 
`{
sd: VerifiableCredential,
complianceCredential: VerifiableCredential
}`

File: src/components/organisms/AssetContent/MetaFull.tsx [hunk 4]
```diff
@@ -3,11 +3,25 @@ import MetaItem from './MetaItem'
...
+import {
+  IVerifiablePresentation,
+  ServiceMetadataMarket
+} from '../../../@types/MetaData'
 
 export default function MetaFull(): ReactElement {
-  const { ddo, metadata, isInPurgatory, type } = useAsset()
-  const { algorithm } = ddo.findServiceByType('metadata').attributes.main
-
+  const { ddo, type, isServiceSelfDescriptionVerified } = useAsset()
```
Comment (sksadjad) [inline src/components/organisms/AssetContent/MetaFull.tsx:9]: extract this and use the code created in the prev comment

File: src/components/organisms/AssetContent/MetaMain.tsx [hunk 5]
```diff
@@ -20,7 +24,16 @@ export default function MetaMain(): ReactElement {
     verifiedServiceProviderName
   } = useAsset()
   const { web3ProviderInfo } = useWeb3()
-
+  const { attributes } = ddo?.findServiceByType(
```
Comment (sksadjad) [inline src/components/organisms/AssetContent/MetaMain.tsx]: extract this and use the code created in the prev comment

**Review by sksadjad — CHANGES_REQUESTED** [review]
made some remarks

File: k8s/gx-portal-deployment.yaml (hunk 1 above)
Comment (zoemaas) [reply k8s/gx-portal-deployment.yaml]: To be honest, I would agree to have it in a different project if we didn't have to deploy it manually. It makes things harder having to search in a different project files belonging to this one. What do you think @nklomp ?

File: src/components/organisms/AssetContent/MetaFull.tsx (hunk 4 above)
Comment (zoemaas) [reply src/components/organisms/AssetContent/MetaFull.tsx:9]: Fixed

File: src/components/organisms/AssetContent/MetaMain.tsx (hunk 5 above)
Comment (zoemaas) [reply src/components/organisms/AssetContent/MetaMain.tsx]: Fixed!

File: src/components/molecules/AssetTeaser.tsx (hunk 2 above)
Comment (zoemaas) [reply src/components/molecules/AssetTeaser.tsx]: Fixed!

File: k8s/gx-portal-deployment.yaml (hunk 1 above)
Comment (nklomp) [reply k8s/gx-portal-deployment.yaml]: They have to be in a different project. This is code that will need to be merged upstream. Infra specific descriptors should never live directly with source code. Bad idea all around

Comment (nklomp) [reply k8s/gx-portal-deployment.yaml]: I made comments about not using this repo for our infra before...

File: src/components/molecules/FormFields/ServiceSelfDescription.tsx (hunk 3 above)
Comment (nklomp) [reply src/components/molecules/FormFields/ServiceSelfDescription.tsx:43]: No

File: k8s/gx-portal-deployment.yaml (hunk 1 above)
Comment (zoemaas) [reply k8s/gx-portal-deployment.yaml]: Fixed!

File: src/components/molecules/FormFields/ServiceSelfDescription.tsx (hunk 3 above)
Comment (zoemaas) [reply src/components/molecules/FormFields/ServiceSelfDescription.tsx:43]: Support for old structure added in FMA-69

**Comment by sksadjad** [comment]
looks good to me

**Review by sksadjad — APPROVED** [review]
looks good to me

**Pull request merged**